Output variable values used and obfuscate sensitive values (#3138)
hashicorp/waypoint: website/content/docs/waypoint-hcl/variables/input.mdx

The description should concisely explain the purpose of the variable and what
kind of value is expected.

### Sensitive Values

[inpage-sensitive]: #sensitive-values

If set to true, the variable value will be obfuscated in outputs. This should not
be relied on as a security measure on its own, but can be used as a way to hide
literal variable values from logging, CLI output, and the UI. Variable values

> shared with the user via these methods will generate a SHA256 encoded value using
> the server cookie as the salt along with the set value. Therefore the user running an
> operation can verify if the value they expected to be used was used, with the output
> SHA256 encoded value.

Reviewer: ```suggestion
shared with the user via these methods will generate a SHA256 hashed value using
the server cookie as the salt along with the set value. Therefore, the user running an
operation can verify whether the expected value was used by verifying the outputted
SHA256 hashed value.
```
